Startup/setup.py

```diff
@@ -49,9 +49,25 @@ def check_exes(args: Args):
         print('No ffmpeg')
         terminate()
 
-    if args.chunk_method == 'vs_ffms2' and (not find_executable('vspipe')):
-        print('vspipe executable not found')
-        terminate()
+    if args.chunk_method in ['vs_ffms2', 'vs_lsmash']:
+        if not find_executable('vspipe'):
+            print('vspipe executable not found')
+            terminate()
+
+        try:
+            import vapoursynth
+            plugins = vapoursynth.get_core().get_plugins()
+        except:
```

Comment:
```suggestion
        except ModuleNotFoundError:
```
This is to address [PEP 8: E722](https://www.flake8rules.com/rules/E722.html).